src/checkbox/checkbox.component.ts

```diff
@@ -213,33 +184,27 @@ export class Checkbox implements ControlValueAccessor, AfterViewInit {
 	/**
 	 * Emits event notifying other classes when a change in state occurs on a checkbox after a
 	 * click.
-	 * @memberof Checkbox
 	 */
 	@Output() change = new EventEmitter<CheckboxChange>();
 	/**
 	 * Emits event notifying other classes when a change in state occurs specifically
 	 * on an indeterminate checkbox.
-	 * @memberof Checkbox
 	 */
 	@Output() indeterminateChange = new EventEmitter<boolean>();
 
 	/**
 	 * Set to `true` if the input checkbox is selected (or checked).
-	 * @memberof Checkbox
 	 */
 	_checked = false;
 	/**
 	 * Set to `true` if the input checkbox is in state indeterminate.
-	 * @memberof Checkbox
 	 */
 	_indeterminate = false;
 
 	currentCheckboxState: CheckboxState = CheckboxState.Init;
 
 	/**
-	 * Maintains a reference to the view DOM element of the `Checkbox`.
-	 * @type {ElementRef}
-	 * @memberof Checkbox
+	 * Maintains a reference to the view DOM element of the `CheckboxComponent`.
```

Comment:
It's `Checkbox` again 🎉 
```suggestion
	 * Maintains a reference to the view DOM element of the `Checkbox`.
```